README.md

```diff
@@ -92,6 +92,7 @@ nix-shell
 
 # Related libraries and initiatives
 
+* [Clashilator](https://github.com/gergoerdi/clashilator): tooling to integrate Clash with Verilator. Enables fast, multithreaded simulation by compiling Clash designs to C++.
```

Comment:
I guess now I'll have to implement some option to enable multithreaded simulation :sunglasses:

Reply:
Oh lol, isn't that the default for Verilator?